frontend/src/pages/ActivityReport/Pages/activitySummary.js

```diff
@@ -307,4 +311,73 @@ ActivitySummary.propTypes = {
   control: PropTypes.object.isRequired,
 };
 
-export default ActivitySummary;
+const sections = [
```

Comment:
There appears to be the potential for defining the whole form in an object similar to this one and simplifying the ~300 lines of code above, but that would be a bigger project then we have the capacity to tackle at this point.